### DISCUSSION TITLE
A session for this runner already exists

### DISCUSSION THREAD

**Discussion by IbotsArvi** [community/community#27073] [Actions]
<p>We’re using self-hosted runners in our organization added at organizational level and it was working fine. What we wanted to do was moving that runner to a repository level. So i removed the runner at the org level and added the same at repository level.</p>
<p>I removed the configuration at the organization level on the runner using this command <code>./config.cmd remove</code> and configured it at repository level.</p>
<p>But when i try to start the runner, it is throwing the following error. Not sure where am going wrong.</p>
<p>Attached a screenshot for ref.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/f/cfd064dccc6829adb74c3b729566ec8f5481a8eb.png" data-download-href="/uploads/short-url/tEpjK91LrKV6yPw6mKvVEISDSgb.png?dl=1" title="Capture" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/85290811/181101581-1598ba97-706a-4596-8de7-9901ba41cb05.png" alt="Capture" data-base62-sha1="tEpjK91LrKV6yPw6mKvVEISDSgb" width="690" height="356" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cfd064dccc6829adb74c3b729566ec8f5481a8eb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture</span><span class="informations">1079×557 6.43 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yaananth** [comment, marked as answer]
<p>Did <code>remove</code> actually work? It usually means that the runner probably isn’t removed (it might be running a job , in which case we won’t remove it).</p>
<p>Please let us know if you are still seeing this, if so please do share repository and runner name details.</p>
<p>Thank you!</p>
